playwright-e2e/config/config.ts

```diff
@@ -1,34 +1,36 @@
 import dotenv from 'dotenv';
 dotenv.config();
 
-const env = process.env.ENVIRONMENT || 'aat';
+const env = process.env.ENV || 'aat';
 
 // Data which can be reused across multiple tests.
 // Called simply with "import config from '../config';" and then e.g. "config.caseworker.email" in a test.
 // Any new data added below will immediately be available wherever config is imported.
 const configuration = {
 
-  timeout: 
-    process.env.TIMEOUT || 10000,
-
-  run_accessibility: 
-    process.env.TESTS_FOR_ACCESSIBILITY || 'false', 
-
   // URLs
   idamUrl:
     process.env.IDAM_API_URL || `https://idam-api.${env}.platform.hmcts.net`,
 
   manageCaseBaseURL:
-    process.env.MANAGE_CASE_URL || `https://manage-case.${env}.platform.hmcts.net`,
+    process.env.CCD_WEB_URL || `https://manage-case.${env}.platform.hmcts.net`,
```

Comment:
I would read CCD_WEB = CCD Admin Web and not Manage Cases (i.e. ExUI). Do we have to change this?

Reply:
The reason I changed is to line up with the env var setup for preview in the existing Jenkins file. Happy to change, but would require some minor tweaks to the existing Jenkins file setup.